packages/cli/src/cli.js

```diff
@@ -37,6 +42,15 @@ const cli = meow(
     patternplate --port 1338
     ✔ Started on http://localhost:1338
 
+    patternplate --port 1338 --listen 0.0.0.0
+    ✔ Started on http://0.0.0.0:1338
+
+    patternplate --port 1338 --listen 0.0.0.0 \\
```

Comment:
Adapt example according to changes above, shorten all paths/hostnames but make them valid, e.g. `abc.com`, `./private-key.pem`